### PR TITLE
Handle geometry validation errors in UI

### PR DESCRIPTION
**Pull request by tsauerwein**
See https://github.com/camptocamp/c2cgeoportal/pull/1200

### CODE REVIEW

File: core/src/script/CGXP/locale/fr.js [hunk 1]
```diff
@@ -161,7 +161,9 @@ GeoExt.Lang.add("fr", {
         forbiddenText: "Vous n'êtes pas autorisé à réaliser cette action&nbsp;!",
         titleText: "Edition",
         saveServerErrorText: "L'enregistrement a échoué en raison d'une erreur du serveur.",
-        queryServerErrorText: "L'interrogation a échoué en raison d'une erreur du serveur."
+        queryServerErrorText: "L'interrogation a échoué en raison d'une erreur du serveur.",
+        titleValidationErrorText: "Erreur de validation",
+        saveValidationErrorText: "Les géométries ne sont pas valides. Raison: "
```
Comment (asaunier) [inline core/src/script/CGXP/locale/fr.js]: "Raison:" => "Raison :"

Comment (tsauerwein) [inline core/src/script/CGXP/locale/fr.js]: Merci, Alex !

**Comment by asaunier** [comment]
Looks good to me :)

File: core/src/script/CGXP/locale/fr.js [hunk 2]
```diff
@@ -161,7 +161,9 @@ GeoExt.Lang.add("fr", {
         forbiddenText: "Vous n'êtes pas autorisé à réaliser cette action&nbsp;!",
         titleText: "Edition",
         saveServerErrorText: "L'enregistrement a échoué en raison d'une erreur du serveur.",
-        queryServerErrorText: "L'interrogation a échoué en raison d'une erreur du serveur."
+        queryServerErrorText: "L'interrogation a échoué en raison d'une erreur du serveur.",
+        titleValidationErrorText: "Erreur de validation",
+        saveValidationErrorText: "Les géométries ne sont pas valides. Raison : "
```
Comment (sbrunner) [inline core/src/script/CGXP/locale/fr.js]: missing `&nbsp;, see line 161 :-)

**Comment by sbrunner** [comment]
Just a very small comment, otherwise that looks good :-)

**Comment by tsauerwein** [comment]
Thanks, I updated the branch.

**Comment by sbrunner** [comment]
Thanks, merge :-)

**Pull request merged**